modules/installation-aws-ami-stream-metadata.adoc

```diff
@@ -0,0 +1,37 @@
+//TODO: Add the module include to the following assemblies
+//TODO: Create related modules for OpenStack (QCOW2 and Bare Metal ISO)
+
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+
+[id="installation-aws-ami-stream-metadata_{context}"]
+= Accessing {op-system} AMIs with stream metadata
+
+In {product-title}, _stream metadata_ provides standardized metadata about {op-system} in the JSON format and injects the metadata into the cluster. Stream metadata is a stable format that supports multiple architectures and is intended to be self-documenting for maintaining automation.
+
+You can use the `coreos print-stream-json` sub-command of `openshift-install` to access information about the boot images in the stream metadata format. This command provides a method for printing stream metadata in a scriptable, machine-readable format.
+
+For user-provisioned installations, the `openshift-install` binary contains references to the version of {op-system} boot images that are tested for use with {product-title}, such as the AWS AMI.
+
+.Procedure
+
+* To parse the stream metadata from a Go program, use the official `stream-metadata-go` library at https://github.com/coreos/stream-metadata-go. You can also view example code in the library.
+
+* To parse the stream metadata via tooling that uses the programming language's JSON library, such as `jq`:
```

Comment:
This statement is confusing.  `jq` is just a command line utility, not a JSON library.

Looking at Colin's feedback, I think you could say something like this:

```
You can also parse the stream metadata using the JSON library of your preferred programming language or via a command line utility that handles JSON data like `jq`.
```

Reply:
Thanks, I was struggling with the wording because as a procedure, I wanted to make it task-based ("Do this" as opposed to "you can do x or y"). Reworded a bit so that hopefully it makes more sense and is still is directive enough to be part of a procedure.